webui/react/src/pages/ExperimentList.table.tsx

```diff
@@ -45,6 +45,12 @@ export const columns: ColumnType<ExperimentItem>[] = [
     sorter: true,
     title: 'State',
   },
+  {
+    dataIndex: 'resourcePool',
+    key: 'resourcePool',
+    sorter: true,
```

Comment:
👍🏼  I didn't know about this. I think I have defined my own alphanumeric sorter for similar usecases before.